redaxo/src/core/lib/console/setup/check.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+/**
+ * @package redaxo\core
+ *
+ * @author staabm
+ *
+ * @internal
+ */
+class rex_command_setup_check extends rex_console_command
+{
+    protected function configure()
+    {
+        $this
+            ->setDescription('Check the commandline interface (CLI) environment for REDAXO requirements')
+        ;
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        $io = $this->getStyle($input, $output);
+        
+        $errors = rex_setup::checkEnvironment();
+        if (count($errors) == 0) {
+            $io->success('PHP version ok');
+        }
+        
+        $res = rex_setup::checkFilesystem();
+        if (count($res) > 0) {
+            $base = rex_path::base();
+            foreach ($res as $key => $messages) {
+                if (count($messages) > 0) {
+                    $affectedFiles = [];
+                    foreach ($messages as $message) {
+                        $affectedFiles[] = str_replace($base, '', $message);
+                    }
+                    $errors[] = rex_i18n::msg($key) . "\n". implode("\n", $affectedFiles);

```

Comment:
an dieser stelle hier `rex_i18n::msg($key)` ist es vermutlich aufwändiger hard-codierte strings zu benutzen als die die hier schon zurückkommen.. oder siehst du eine andere möglichkeit @gharlan  ?

Reply:
Das meinte ich ja mit dem dritten Punkt in den "Regeln". Da wir hier schon Messages aus der Funktion bekommen, können die natürlich auch verwendet werden. 
Also so lassen 👍